verification/spec/connection-handshake/ch.tla

```diff
@@ -0,0 +1,188 @@
+--------------------------------- MODULE CH ---------------------------------
+
+
+VARIABLES
+    parties,            \* The state of each party.
```

Comment:
Maybe better to call it chModule (connectionHandshakeModule)?

Reply:
We also probably want to have in this spec only single module with an environment and then we will compose multi-module system in a separate spec by creating instances of this spec. So it should be a single module. In that case we don't need a variable for a party (module), but instead directly module state. You can start with connectionEnd field and a connectionProof. And we will probably need also lightClient with a proof.